extractor/Dockerfile-porter

```diff
@@ -11,7 +11,7 @@ COPY ./pyproject.toml /src/pyproject.toml
 COPY ./poetry.lock /src/poetry.lock
 
 # Install dependencies
-RUN poetry config virtualenvs.create false && poetry install --no-interaction --no-ansi
```

Comment:
Can we remove the porter Dockerfiles ? 
We have them for extractor and backend but not for ai-hub, it's confusing